src/setup-elixir.js

```diff
@@ -103,15 +134,14 @@ function getVersionFromSpec(spec, versions) {
   }
 }
 
-async function getOtpVersions(osVersion) {
-  const result = await get(
-    `https://repo.hex.pm/builds/otp/${osVersion}/builds.txt`
-  )
+async function getOTPVersions(osVersion) {
+  const result = await get(`https://repo.hex.pm/builds/otp/${osVersion}/builds.txt`,
+    'otp_builds.txt')
```

Comment:
What is the purpose of the separate test file?

Reply:
`builds.txt` lists Elixir versions, `otp_builds.txt` lists Erlang versions. They come from different sources in Hex.pm. I don't understand the question, probably. If the input is different (`builds.txt` for Elixir v. `builds.txt` from Erlang) should the tests', also?

Reply:
Does this make sense? Can I Resolve it, or do you want some type of change?

Reply:
I'll make this more explicit by renaming the files.

Reply:
I am wondering why we need a separate test file when we can fetch it from the source repo.hex.pm. I would like to avoid mixing test and production code if possible.

Reply:
So... we're fetching versions dynamically, right? And then we have expressions like ^1.10, which will resolve to the latest published (I got this error yesterday, which is why I fixed the tests to use `elixir_builds.txt`). If we depend on an external source, there's less correctness in repeatability (i.e. you don't know how/when a change in an external resource will affect you). I don't mind moving the code away from it, though (e.g. by exposing the route as a parameter instead of juggling env. variables).

Reply:
Production code is separated from test code as much as possible.